metagraph/CMakeLists.txt

```diff
@@ -331,8 +334,18 @@ add_library(mersenne_twister STATIC
 add_library(sdust STATIC
   external-libraries/sdust/sdust.c
 )
+
 target_compile_options(sdust PRIVATE -Wall -Wc++-compat -Wno-error)
 
+include(ExternalProject)
+ExternalProject_Add(HTSLIB_Project
+  SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/external-libraries/htslib
+  CONFIGURE_COMMAND autoreconf -i ${CMAKE_CURRENT_SOURCE_DIR}/external-libraries/htslib && ${CMAKE_CURRENT_SOURCE_DIR}/external-libraries/htslib/configure --prefix=${CMAKE_CURRENT_SOURCE_DIR}/external-libraries/htslib --disable-libcurl --disable-bz2 --disable-lzma
```

Comment:
Why no bz2?

Reply:
It was having linking errors at some point, but I'll try it again now to see if it works.

Reply:
So it seems like it fails to link if I enable BZ2. According to HTSlib's documentation, bz2 is only used for loading cram files, so I think it should be fine to omit this.